app/models/ontology/symbols.rb

```diff
@@ -44,8 +44,8 @@ def update_or_create_from_hash(hash, timestamp = Time.now)
             if e.label
               e.label
             else
-              portion = e.name.split('#', 2).last
-              portion.end_with?('>') ? portion[0..-2] : portion
+              portion = e.name.end_with?('>') ? e.name[0..-2] : e.name
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.